tests/foreman/ui/test_ansible.py

```diff
@@ -298,28 +298,8 @@ def test_positive_role_variable_information(self):
     """
 
 
-@pytest.mark.stubbed
 @pytest.mark.tier2
-def test_positive_assign_role_in_new_ui(self):
-    """Using the new Host UI, assign a role to a Host
-
-    :id: 044f38b4-cff2-4ddc-b93c-7e9f2826d00d
-
-    :steps:
-        1. Register a RHEL host to Satellite.
-        2. Import all roles available by default.
-        3. Navigate to the new UI for the given Host.
-        4. Select the 'Ansible' tab
-        5. Click the 'Assign Ansible Roles' button.
-        6. Using the popup, assign a role to the Host.
-
-    :expectedresults: The Role is successfully assigned to the Host, and shows up on the UI
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier2
-def test_positive_remove_role_in_new_ui(self):
+def test_positive_add_and_remove_ansible_role_in_new_ui(target_sat, function_host):
```

Comment:
Looks like you are merging these 2 stubbed tests
```suggestion
def test_positive_assign_and_remove_ansible_role_in_new_ui(target_sat, function_host):
```